dashboard/test/controllers/api/v1/teacher_feedbacks_controller_test.rb

```diff
@@ -25,6 +25,13 @@ class Api::V1::TeacherFeedbacksControllerTest < ActionDispatch::IntegrationTest
     assert_equal @teacher.id, teacher_feedback.teacher_id
   end
 
+  test 'retrieves no content when no feedback is available' do
+    sign_in @teacher
+    get "#{API}/get_feedback_from_teacher", params: {student_id: @student.id, level_id: @level.id, teacher_id: @teacher.id}
+
+    assert_response 204
```

Comment:
nit: maybe `:no_content` here instead of 204 for consistency? Not a big deal either way